README.md

```diff
@@ -18,12 +20,53 @@ Some layers include the corresponding OTel language SDK for the Lambda. This all
 * ### [.NET + Collector Lambda Layer](dotnet/README.md)
 * ### [Collector Lambda Layer](collector/README.md)
 
-
 ## FAQ
+
 * **What exporters/recievers/processors are included from the OpenTelemetry Collector?**
     > You can check out [the stripped-down collector's imports](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L18) in this repository for a full list of currently included components.
 * **Is the Lambda layer provided or do I need to build it and distribute it myself?**
     > This repository does not provide pre-build Lambda layers. They must be built manually and saved in your AWS account. This repo has files to facilitate doing that. More information is provided in [the Collector folder's README](collector/README.md).
 
 ## Design Proposal
+
 To get a better understanding of the proposed design for the OpenTelemetry Lamda extension, you can see the [Design Proposal here.](docs/design_proposal.md)
+
+## Features
+
+The following is a list of features provided by the OpenTelemetry layers.
+
+### Custom Context Propagation
+
+Context can be propagated through various mechanisms (e.g. http headers (APIGW), message attributes (SQS), ...). In some cases, it may be required to pass a custom context propagation extractor in lambda through configuration, this feature allows this through Lambda instrumentation configuration.
+
+### Disable AWS Context propagation via environment variable
+
+The context propagation when Active Tracing is enabled uses the `_X_AMZN_TRACE_ID` environment variable present in the Lambda runtime. To disable this behaviour, implementations have added the `OTEL_LAMBDA_DISABLE_AWS_CONTEXT_PROPAGATION` environment variable which can be set to `true`.
```

Comment:
It is not, there are two language implementations I'm aware of that are close to or have added it: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1227 https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1507

Reply:
I think we should  change this to be an option to link a context extracted from the Lambda runtime environment to the instrumentation-generated span rather than disabling that context extraction entirely.

Reply:
@Aneurysm9 could this be documented in a separate line? i'd like to capture what's there today

Reply:
I've added a separate line for span link context propagation

Reply:
(FWIW, I'm also in favor of removing the need for the setting by shifting the env var context to a span link instead of formalizing a setting for disabling it. Documenting where the setting is currently implemented is fine though.)

Reply:
I've removed the line for disabling context propagation via env variable.